aws/resource_aws_route53_health_check_test.go

```diff
@@ -33,17 +36,14 @@ func TestAccAWSRoute53HealthCheck_basic(t *testing.T) {
 			},
 			{
 				Config: testAccRoute53HealthCheckConfigUpdate,
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckRoute53HealthCheckExists(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "failure_threshold", "5"),
-					resource.TestCheckResourceAttr(resourceName, "invert_healthcheck", "false"),
-				),
+				PlanOnly: true,
```

Comment:
This test changes values, so leaving it as a `Check` would be good.

Reply:
oops, didnt mean to change that.